src/content/docs/en/guides/deploy/space.mdx

```diff
@@ -0,0 +1,122 @@
+---
+title: Deploy your Astro Site to Space
+description: How to build an Astro site in Space.
+type: deploy
+i18nReady: true
+---
+import InstallGuideTabGroup from '~/components/TabGroup/InstallGuideTabGroup.astro';
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+[Deta Space](https://deta.space/developers) is a personal computer that lives in the cloud — a ‘personal cloud’. You can build and run apps for yourself in your own "Space". You can publish the apps you've built, and they'll run for people all around the world.
+
+This guide includes step-by-step instructions for building sites in Space. Both static and server-side rendered Astro sites are supported.
+
+## Prerequisite
+
+To push an Astro site to Space, make sure you first:
+
+- Create a [Space account](https://deta.space).
+- Install the [Space CLI](https://deta.space/docs/en/basics/cli) and log in.
+
+Create a Space project **inside** the directory of your Astro project. Run the CLI and follow the instructions on the screen.    
+
+```bash
+space new
+```
+
+:::note
+The Space CLI will try to detect the right configuration for your app. Accept the suggested configuration and follow the instructions below on how to configure it for your Astro app.
+:::
+
+
+## Project Configuration
+
+### Static site
+
+If you build a simple Astro site with no adapters, you don’t need any extra configuration to your Astro project. 
+
+Edit the `Spacefile` file generated by the Space CLI on the root of your project. You will need to change the engine to `static`, add the `build` command, and serve the `dist` directory generated by Astro. 
+
+```yaml title="Spacefile" "static" "- npm run build" "serve:dist"
```

Comment:
The `serve:dist` is missing a space for the highlight to work:

```suggestion
```yaml title="Spacefile" "static" "- npm run build" "serve: dist"
```